app/controllers/editions_controller.rb

```diff
@@ -405,6 +408,11 @@ def tagging_update_form_params
   end
 
   def format_failure_message(resource)
+    if resource.instance_of?(LocalTransactionEdition) && resource.invalid?
+      resource.errors[:base] << resource.errors.full_messages
+      resource.errors[:base].flatten!
+    end
```

Comment:
It would be good to discuss this.  What I found was that model validation errors weren't being displayed at all - the field is highlighted (on the JS disabled version - but not on the JS enabled version) and there aren't messages displayed in the context of the label/field itself.